nestkernel/model_manager.cpp

```diff
@@ -492,17 +504,18 @@ ModelManager::calibrate( const TimeConverter& tc )
 bool
 ModelManager::compare_model_by_id_( const int a, const int b )
```

Comment:
Is the name of this method right? We pass in ints, presumably model IDs, but the sorting criterium is the name.

Reply:
A better name would be `compare_models_given_by_id_by_their_name`, but I'd rather not touch this right now. I actually think the whole `memory_info()` thing together with the custom allocator for models should go away. I reckon that a modern allocator would easily outperform our 18 year old custom solution...

Reply:
OK!